src/gpu-compute/compute_unit.cc

```diff
@@ -106,6 +107,61 @@ ComputeUnit::ComputeUnit(const Params &p) : ClockedObject(p),
     scalar_resp_tick_latency(
             p.scalar_mem_resp_latency * p.clk_domain->clockPeriod()),
     memtime_latency(p.memtime_latency * p.clk_domain->clockPeriod()),
+    mfma_scale(p.mfma_scale),
+    mfma_cycles({
+        {GfxVersion::gfx90a, {
```

Comment:
please add a comment here identifying GFX90A as MI200 and gfx942 as MI300

Reply:
I would additionally point to the ISA specs and mention that is where all of these numbers came from.  The table for MI300 and the "passes" in the descriptions for MI200

Reply:
done